src/util/util.js

```diff
@@ -24,14 +26,26 @@ function getBlockTotalFees(coinbaseTx, blockHeight) {
 
   var blockReward = currentBlockReward(blockHeight);
 
-  let totalOutput = 0;
+  const inputs = coinbaseTx.inputs;
+  const outputs = coinbaseTx.outputs;
 
-  for (let output of coinbaseTx.outputs) {
-    totalOutput += output.value;
+  // Start with the required coinbase output
+  let minerOutput = outputs[0].value;
+
+  for (let i = 1; i < outputs.length; i++) {
+    // Miner added extra subsidy payout outputs
+    if (!inputs[i]) {
+      minerOutput += outputs[i].value;
+    } else {
+      // let proof;
```

Comment:
I think you want to use `output.covenant.isClaim` on the linked output to know if its an `AirdropProof` or an `OwnershipProof`

Reply:
These lines aren't about determining which kind of proof it is, but actually decoding the proof. For example, both type of proofs have a `.fee` value which might be interesting to display on their own. It's not necessary for this quick total fees calculation but I left the comments in there in case we want to fancy it up.

Reply:
Definitely want to fancy this up at some point, but for now I think this perfect. Putting on my triage hat, and going to try to get this explorer to a livable standard before the next testnet hits, and then hopefully put out a roadmap to hit some of the more fine grained details.